x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -658,4 +658,37 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.memory_protection.shellcode_enhanced_pe_parsing',
+    first_supported_version: '7.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.memory_protection.shellcode_enhanced_pe_parsing',
+      {
+        defaultMessage:
+          "A value of 'false' disables enhanced parsing of PEs found within shellcode payloads. Default: true.",
+      }
+    ),
+  },
+  {
+    key: 'windows.advanced.memory_protection.shellcode',
+    first_supported_version: '7.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.memory_protection.shellcode',
+      {
+        defaultMessage:
+          "A value of 'false' disables Shellcode Injection Protection, a feature of Memory Protection. Default: true.",
+      }
+    ),
+  },
+  {
+    key: 'windows.advanced.memory_protection.memory_scan',
+    first_supported_version: '7.15',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.memory_protection.signature',
```

Comment:
Is part of the string here supposed to be identical to the `key`? The two above two look like they are.

Reply:
I guess is a miss-match. Somehow the key is named slightly different than the text it shows. I tried to write the translation key close to the text and not the key.